master/docs/manual/configuration/changesources.rst

```diff
@@ -910,8 +910,9 @@ It accepts the following arguments:
     Specifies public host key to match when authenticating with SSH
     public key authentication. This may be either a :ref:`Secret` or just a
     string. `sshPrivateKey` must be specified in order to use this option.
-    The host key must be in the form of `<key type> <base64-encoded string>`,
-    e.g. `ssh-rsa AAAAB3N<...>FAaQ==`.
+    The host key must be either bare key (usually ``<key type> <base64-encoded string>``, e.g.
+    e.g. ``ssh-rsa AAAAB3N<...>FAaQ==``.), or contents of known_hosts file that will be passed to
```

Comment:
Doubled "e.g." here plus extra linebreak inserted compared to docs in buildsteps.rst.

Reply:
Fixed now.